libcxx/include/__expected/expected.h

```diff
@@ -1145,8 +1145,8 @@ class expected : private __expected_base<_Tp, _Err> {
     requires(!is_void_v<_T2>)
 #  if _LIBCPP_STD_VER >= 26
             && requires {
-                 { *__x == *__y } -> __core_convertible_to<bool>;
-                 { __x.error() == __y.error() } -> __core_convertible_to<bool>;
+                 { *__x == *__y }->__core_convertible_to<bool>;
+                 { __x.error() == __y.error() }->__core_convertible_to<bool>;
```

Comment:
Let's avoid pure unnecessary (?) formatting changes.

Also, I think it's also unnecessary to refer to other PRs in the PR description, which will make the by-default squashed commit message to refer to them and show the them several times on GitHub UI. Those PRs are arguably orthogonal to adding `[[nodiscard]]`.